add swarm video hashes

tmm360 opened the pull request.

tmm360 committed:
add swarm video hashes

ligi commented:
@tmm360 why do I get a 404 when I try to access these swarm hashes on a gateway? e.g.:
https://gateway.ethswarm.org/access/d2fa049d664484b158c36db2c05b9ff461267f4ee44787a45a7ba182dabe07fc

![image](https://github.com/user-attachments/assets/fac0963b-b97b-4fe4-b3b8-1bb42474110f)

tmm360 commented:
Hi @ligi, thanks for your question!
This is due to an issue that we have encountered trying to publish contents on bee nodes: https://github.com/ethersphere/bee/issues/4864

As you can see, at moment it is still problematic to keep data alive when working at low level with chunks, as we are currently doing. Simply because of some client API design issues.

To work around this, and have contents published in time for devcon, I've implemented a reverse proxy with cache for swarm chunks, and in this way I've replaced the problematic pinning component of Bee.

Said this, the work has been done very fast before devcon (issue has been created on October 12th), contents have been chunked with swarm protocol, postage batches (swarm space rental) have been purchased on swarm net, but data hasn't still been pushed on swarm network for real. We are working on this, and it should be done by the end of this year.

So this is the reason why you can't still found videos on swarm network. We have now more than 1TB of data chunks to push, but routines to do this still need to be completed.
